data/korath/korath variants.txt

```diff
@@ -413,6 +413,43 @@ ship "Korath Raider Mark XLII" "Korath Raider Mark XLII (Crippler)"
 	turret "Korath Banisher"
 
 
+ship "Korath Ravager" "Korath Ravager (Crippler)"
+	outfits
+		"Korath Detainer" 2
+		"Korath Repeater Turret" 2
+		"Korath Warder"
+		"Korath Piercer Launcher" 2
+		"Korath Piercer" 62
+		"Firelight Missile Bank" 4
+		"Firelight Missile" 80
+
+		"Systems Core (Large)"
+		"Fuel Processor" 2
+		"Plasma Core"
+		"Triple Plasma Core"
+		"Liquid Sodium Cooler"
+		"Small Heat Shunt"
+		"Outfits Expansion"
+		"Korath Repeater Rifle" 219
+
+		"Afterburner (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Lunar Class)"
+		"Jump Drive"
+
+	gun "Korath Detainer"
+	gun "Korath Detainer"
+	gun "Korath Piercer Launcher"
+	gun "Korath Piercer Launcher"
+	gun "Firelight Missile Bank"
+	gun "Firelight Missile Bank"
+	gun "Firelight Missile Bank"
+	gun "Firelight Missile Bank"
+	turret "Korath Repeater Turret"
+	turret "Korath Repeater Turret"
+	turret "Korath Warder"
+
+
```

Comment:
Very optional, if you want to keep the inferno variant, have something like this pop up only post-automata along with the crippler (as opposed to the heaver being a remnant/exile unlock)

```suggestion
 ship "Korath Ravager" "Korath Ravager (Inferno)"
 		"required crew" 189
		"bunks" 199
		"weapon capacity" 444
	outfits
		"Korath Inferno" 2
		"Korath Warder"
		"Korath Fire-Lance" 2
		"Firelight Missile Bank" 6
		"Firelight Missile" 120

		"Systems Core (Large)"
		"Fuel Processor" 2
		"Plasma Core"
		"Triple Plasma Core"
		"Liquid Sodium Cooler"
		"Korath Repeater Rifle" 219

		"Afterburner (Planetary Class)"
		"Steering (Lunar Class)"
		"Thruster (Planetary Class)"
		"Jump Drive"
	
	gun "Korath Fire-Lance"
	gun "Korath Fire-Lance"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	turret "Korath Inferno"
	turret "Korath Inferno"
 	turret "Korath Warder"
 

```

Reply:
I'm not too worried about it for now. We might balance it in the future with additional things like hull and shields, but I think I'm okay with saving this for later (likely with the Firestorm Version depending on when Stage II is merged). Thank you though.